src/main/java/org/terasology/launcher/game/GameDownloader.java

```diff
@@ -57,7 +57,18 @@ public GameDownloader(File downloadDirectory, File tempDirectory, boolean saveDo
         if (downloadZipFile.exists() && (!downloadZipFile.isFile() || !downloadZipFile.delete())) {
             throw new IOException("Could not delete file! " + downloadZipFile);
         }
-        downloadURL = DownloadUtils.createFileDownloadUrlJenkins(jobName, buildNumber, DownloadUtils.FILE_TERASOLOGY_GAME_ZIP);
+
+        // If we have a matching Omega distribution for this game version then fetch that zip file instead
+        if (gameVersion.getOmegaNumber() != null) {
+            logger.info("Omega distribution {} is available for that engine build, downloading it", gameVersion.getOmegaNumber());
+            downloadURL = DownloadUtils.createFileDownloadUrlJenkins(gameVersion.getJob().getOmegaJobName(),
+                    gameVersion.getOmegaNumber(), DownloadUtils.FILE_TERASOLOGY_OMEGA_ZIP);
+        } else {
+            logger.warn("Engine build {} has no Omega zip available! Falling back to main zip without extra modules", buildNumber);
```

Comment:
Could this be a problem? As in, the game suddleny loses lots of modules and saves don't work anymore?